src/librustc_trans/abi.rs

```diff
@@ -139,7 +139,8 @@ impl ArgAttributes {
 pub enum RegKind {
     Integer,
     Float,
-    Vector
+    Vector,
+    X86Mmx,
```

Comment:
Same here (`x86_mmx: bool` inside `Vector`), with the caveat that the system here is subject to change/removal in the future.